NOTICE-binary

```diff
@@ -28,6 +28,7 @@ Copyright 2014-2019 The Apache Software Foundation
 
 This project bundles the following dependencies under the Apache Software License 2.0 (http://www.apache.org/licenses/LICENSE-2.0.txt)
 
+- com.data-artisans:frocksdbjni:5.17.2-artisans-1.0
```

Comment:
Do we need to replace `data-artisans` with `ververica`?

Reply:
No, I think frocks is released under `com.data-artisans`